app/views/components/sequence_checkbox.scala.html

```diff
@@ -28,12 +28,34 @@
     }
 }
 
-@itemChecked(idx: Int) = @{form.data.valuesIterator.exists(_ == idx.toString)}
+@hintClass = @{
+  if(option.exclusive) {
+    "exclusive-hint"
+  } else {
+    s"checkbox-hint-$idx"
+  }
+}
+
+@ariaDescribed = @{
+    if (option.hint) {
+        val dataExclusive = if (option.exclusive) {
+            "data-exclusive='true'"
+        } else {
+            ""
+        }
+
+        Html(dataExclusive + s"aria-describedby='$hintClass'")
+    } else {
+        Html(None)
+    }
+}
+
+@itemChecked(idx: Int) = @{form.data.valuesIterator.contains(idx.toString)}
```

Comment:
this is generating 
<input class="govuk-checkboxes__input" id="start-3" name="start[3]" type="checkbox" value="3"  data-exclusive='true'aria-describedby='exclusive-hint'>

note no space before aria-describedby, also better to remain consistent in quoting with "" rather than ''

original line 
<input class="govuk-checkboxes__input" id="current-3" name="current[3]" type="checkbox" value="3"   data-exclusive="true" aria-describedby="exclusive-hint">

Reply:
Tricky to fix the quotations but I can try.

Reply:
you can use view.components.ariaDescribedBy.scala.html I think